modules/templates-writing-parameters.adoc

```diff
@@ -63,13 +63,19 @@ characters long consisting of all upper and lowercase alphabet letters
 and numbers.
 
 The syntax available is not a full regular expression syntax. However, you can
-use `\w`, `\d`, and `\a` modifiers:
+use `\w`, `\d`, `\a`, and `\A` modifiers:
 
 - `[\w]{10}` produces 10 alphabet characters, numbers, and underscores. This
 follows the PCRE standard and is equal to `[a-zA-Z0-9_]{10}`.
 - `[\d]{10}` produces 10 numbers. This is equal to `[0-9]{10}`.
 - `[\a]{10}` produces 10 alphabetical characters. This is equal to
 `[a-zA-Z]{10}`.
+- `[\A]{10}` produces 10 punctuation or symbol characters. This is equal to ``[~!@#$%\^&*()\-_+={}\[\]\\|<,>.?/"';:` ]{10}``.
++
+[NOTE]
+====
+In the example, the `\`, `^`, `-`, `[`, and `]` characters are used as escape characters. The `\A` parameter would be any of the following characters: ``~!@#$%^&*()-_+={}[]\|<,>.?/"';:` ``. Depending on if the template is written in YAML or JSON, the backslash `\` may need to be escaped with a second backslash `\\`.
```

Comment:
I'm having a hard time understanding "... characters are used as escape characters". How about changing it to "... characters are escaped so that they lose the special meaning that they would normally have within a bracketed character class".

And instead of "The `\A` parameter would be any of the following characters" it would be clearer to write "The `\A` modifier produces one of the following characters"; the modifier/producer terms are consistent with the main text of the documentation.

I think the note about escaping the `\` is best put in a separate note, or otherwise, elsewhere. It's certainly useful, but it apples to all the modifiers, not just the `\A` modifier.